### PR TITLE
fix(fetched item): FIxed overlaping text when someone fetch item

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
SD-2030

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: client/app/scripts/superdesk-archive/views/fetched-desks.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div ng-if="desks.length">
-	<span translate>fetched in:</span>
-	<span ng-repeat="desk in desks"><a ng-href="">{{desk.desk.name}} ({{desk.count}})</a>{{$last ? '' : ', '}} </span>
-</div>
+<dl ng-if="desks.length">
+	<dt translate>fetched in:</dt>
+	<dd ng-repeat="desk in desks"><a ng-href="">{{desk.desk.name}} ({{desk.count}})</a>{{$last ? '' : ', '}} </dd>
```
Comment (plamut) [inline client/app/scripts/superdesk-archive/views/fetched-desks.html]: I recommend adding `track by` here.

**Comment by amagdas** [comment]
Is this going to be updated?

**Comment by plamut** [comment]
I would say yes (@pavlovicnemanja92 ?), it's just that yesterday was totally packed with meetings and there was almost no time for any development.

**Comment by plamut** [comment]
:+1:

**Pull request merged**